### PR TITLE
Lfa 554765

### PR DESCRIPTION
**Pull request by lfasani**

### CODE REVIEW

**Comment by sandupostaru** [comment]
Both tests are ok:
* https://ci.eclipse.org/capella/job/capella-product/job/PR-98/2/testReport/org.polarsys.capella.test.diagram.misc.ju.testcases/DeleteFromModelContainerSemanticTarget/
* https://ci.eclipse.org/capella/job/capella-product/job/PR-98/2/testReport/org.polarsys.capella.test.diagram.misc.ju.testcases/DeleteFromModelSemanticTarget/

**Comment by sandupostaru** [comment]
I think we should squash the 3 commits into one, what do you think @pdulth ?

File: tests/plugins/org.polarsys.capella.test.diagram.misc.ju/src/org/polarsys/capella/test/diagram/misc/ju/testcases/delete/AbstractDeleteFromModelTestCase.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package org.polarsys.capella.test.diagram.misc.ju.testcases.delete;
```
Comment (pdulth) [inline tests/plugins/org.polarsys.capella.test.diagram.misc.ju/src/org/polarsys/capella/test/diagram/misc/ju/testcases/delete/AbstractDeleteFromModelTestCase.java]: copyright

Comment (sandupostaru) [reply tests/plugins/org.polarsys.capella.test.diagram.misc.ju/src/org/polarsys/capella/test/diagram/misc/ju/testcases/delete/AbstractDeleteFromModelTestCase.java]: :(

**Pull request merged**